
From: "Luiz Fernando N. Capitulino" <lcapitulino@prefeitura.sp.gov.br>

drivers/net/ne2.c does not compile without CONFIG_MCA_LEGACY set.

As CONFIG_MCA_LEGACY depends on CONFIG_MCA, we can use only
CONFIG_MCA_LEGACY, insteed of "MCA && MCA_LEGACY".

Signed-off-by: Luiz Capitulino <lcapitulino@prefeitura.sp.gov.br>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/net/Kconfig |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN drivers/net/Kconfig~net-ne2c-needs-mca_legacy drivers/net/Kconfig
--- 25/drivers/net/Kconfig~net-ne2c-needs-mca_legacy	2004-06-21 23:18:47.021100192 -0700
+++ 25-akpm/drivers/net/Kconfig	2004-06-21 23:18:47.034098216 -0700
@@ -1122,7 +1122,7 @@ config SKMC
 
 config NE2_MCA
 	tristate "NE/2 (ne2000 MCA version) support"
-	depends on NET_ETHERNET && MCA
+	depends on NET_ETHERNET && MCA_LEGACY
 	select CRC32
 	help
 	  If you have a network (Ethernet) card of this type, say Y and read
_
